sdk/data/azappconfig/client.go

```diff
@@ -139,19 +138,10 @@ func (c *Client) GetSetting(ctx context.Context, key string, options *GetSetting
 		return GetSettingResponse{}, err
 	}
 
-	var lastModified *time.Time
-	if resp.LastModified != nil {
-		tt, err := time.Parse(http.TimeFormat, *resp.LastModified)
-		if err != nil {
-			return GetSettingResponse{}, err
-		}
-		lastModified = &tt
-	}
-
 	return GetSettingResponse{
 		Setting:      settingFromGenerated(resp.KeyValue),
 		SyncToken:    SyncToken(*resp.SyncToken),
-		LastModified: lastModified,
+		LastModified: resp.LastModified,
```

Comment:
So this is interesting. Originally, `GetSettingResponse.LastModified` was populated from the `Last-Modified` header value which is no longer the case. Maybe it was duplicate data? If the value of `GetSettingResponse.KeyValue.LastModified` is the same, then your change is fine even though it's not technically accurate (i.e. it's not coming from the `Last-Modified` header). Assuming that's the case, this should be slightly changed.
```go
LastModified: resp.KeyValue.LastModified
```

Reply:
yes. This is right. Updating